acropolis-persistence/src/main/kotlin/org/ephyra/acropolis/persistence/api/persistence/NetworkPersistence.kt

```diff
@@ -0,0 +1,30 @@
+package org.ephyra.acropolis.persistence.api.persistence
+
+import org.ephyra.acropolis.persistence.api.entity.NetworkEntity
+import org.ephyra.acropolis.persistence.impl.NetworkRepository
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+@Component
+class NetworkPersistence {
+    @Autowired
+    private lateinit var repo: NetworkRepository
+
+    fun create(network: NetworkEntity) {
+        repo.save(network)
+    }
+
+    fun getNetwork(networkId: Long, projectId: Long): NetworkEntity? {
```

Comment:
Doesn't match the rest of the persistence core. Should be just `find`.

Reply:
Fixed